Source: written by Pierre Lejeune, for Exherbo
Upstream: no
Reason: Various fixes in Makefile
--- Makefile.orig	2010-04-07 12:03:49.000000000 +0200
+++ Makefile	2010-11-23 21:57:34.661127516 +0100
@@ -81,7 +81,7 @@
 # install directories
 INSTALLPREFIX ?= $(PREFIX)/usr
 INSTALLDIRINC = $(INSTALLPREFIX)/include/miniupnpc
-INSTALLDIRLIB = $(INSTALLPREFIX)/lib
+INSTALLDIRLIB = $(INSTALLPREFIX)/$(LIBDIR)
 INSTALLDIRBIN = $(INSTALLPREFIX)/bin
 
 FILESTOINSTALL = $(LIBRARY) $(EXECUTABLES)
@@ -148,16 +148,16 @@
 
 $(SHAREDLIBRARY):	$(LIBOBJS)
 ifeq ($(OS), Darwin)
-	$(CC) -dynamiclib $(LDFLAGS) -Wl,-install_name,$(SONAME) -o $@ $^ 
+	$(CC) $(CFLAGS) $(LDFLAGS) -dynamiclib -Wl,-install_name,$(SONAME) -o $@ $^ 
 else
-	$(CC) -shared $(LDFLAGS) -Wl,-soname,$(SONAME) -o $@ $^
+	$(CC) $(CFLAGS) $(LDFLAGS) -shared -Wl,-soname,$(SONAME) -o $@ $^
 endif
 
 upnpc-static:	upnpc.o $(LIBRARY) $(LDLIBS)
-	$(CC) $(LDFLAGS) -o $@ $^
+	$(CC) $(CFLAGS) $(LDFLAGS) -o $@ $^
 
 upnpc-shared:	upnpc.o $(SHAREDLIBRARY) $(LDLIBS)
-	$(CC) $(LDFLAGS) -o $@ $^
+	$(CC) $(CFLAGS) $(LDFLAGS) -o $@ $^
 
 testminixml:	$(TESTMINIXMLOBJS)
 
